src/models/BridgeRequest.ts

```diff
@@ -37,8 +38,10 @@ export class BridgeRequest {
     reject(err?: unknown) {
         this.req.reject(err);
     }
+}
 
-    public static ERR_VIRTUAL_USER = "virtual-user";
-    public static ERR_NOT_MAPPED = "virtual-user";
-    public static ERR_DROPPED = "virtual-user";
+export enum BridgeRequestErr {
```

Comment:
We're using Typescript `enum`s here instead of string values because they are quite handy. Similar to C# enums.